docs/resources/aviatrix_controller_access_allow_list_config.md

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Settings"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_controller_access_allow_list_config"
+description: |-
+  Creates and manages an Aviatrix Controller's Access Allow List  Config
+---
+
+# aviatrix_controller_access_allow_list_config
+
+!> **WARNING:** If any of the Allow List IPs are incorrect, the Controller will be inaccessible.
+
+The **aviatrix_controller_access_allow_list_config** resource allows management of an Aviatrix Controller's Access Allow List.
+
+## Example Usage
+
+```hcl
+# Create an Aviatrix Controller Access Allow List Config
+resource "aviatrix_controller_access_allow_list_config" "test" {
+  allow_list {
+    ip_address = "0.0.0.0"
+  }
+
+  allow_list {
+    ip_address = "1.2.3.4"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+### Required
+* `allow_list` - (Required) List of allowed IPs.
+  * `ip_address` - (Required) IP address.
+  * `description` - (Optional) Description.
```

Comment:
Can we give a more detailed description for these three attributes, including in the resource.